public/index.html

```diff
@@ -16,6 +16,11 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+    href="https://fonts.googleapis.com/css2?family=Inter:wght@600&family=Roboto:wght@300;400;500;700&display=swap"
+    rel="stylesheet"
```

Comment:
check if these links are correct, there is no closing tag for last link element and crossorigin attribute has no value

Reply:
@pashedin  crossorigin is like that only. There is closing tag below but adding that

Reply:
@pashedin added the closed link tag and commited